README.md

```diff
@@ -122,19 +126,25 @@ _You may also use a symlink as an alternative to copying: `ln -s sample.env .env
 4. **Start the Services**
 
 ```bash
-./manage.sh -n <network> -a start
+./manage.sh -n <network> -a up
 ```
 
 - With optional proxy:
 
 ```bash
-./manage.sh -n <network> -a start -f proxy
+./manage.sh -n <network> -a up -f proxy
+```
+
+- With optional bitcoin node:
+
+```bash
+./manage.sh -n <network> -a up -f bitcoin
 ```
 
 5. **Stop the Services**
 
 ```bash
-./manage.sh -n <network> -a stop
+./manage.sh -n <network> -a down
```

Comment:
technically, this is still correct - but using the sysv standard `stop` is more accurate

Reply:
explained on the comment above. We can continue the conversation there.